winml/lib/Telemetry/WinMLTelemetryHelper.cpp

```diff
@@ -16,6 +16,22 @@ WinMLTelemetryHelper::WinMLTelemetryHelper()
 WinMLTelemetryHelper::~WinMLTelemetryHelper() {
 }
 
+void WinMLTelemetryHelper::LogApiUsage(const char* name){
+  if (!telemetry_enabled_)
+    return;
+  WinMLTraceLoggingWrite(
+      provider_,
+      "ApiUsage",
+      TraceLoggingKeyword(WINML_PROVIDER_KEYWORD_DEFAULT),
+      TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
+      //Telemetry info
+      TraceLoggingUInt8(WINML_TLM_EXPERIMENTAL_API_VERSION, "experimentalSchemaVersion"),
+      // named dimension override info
```

Comment:
Is this "name" variable referring to the name of the API used or the dimension that is overridden? Maybe the comment should be // api usage info to be less confusing